packages/mobile_app/lib/blocs/user_poll.dart

```diff
@@ -0,0 +1 @@
+export 'package:user_poll/bloc.dart';
```

Comment:
For me as not experienced in Dart person this seems to much wells - `blocks.dart` that exports `user_poll.dart` which exports `blocks.dart`. This is confusing and IMO too much files produced.

Reply:
It is done, if we want to extend our bloc for flutter app. But seems like it will be never used and I must clean it.

Reply:
Done.